packages/ui-react/src/components/Subscription/Subscription.tsx

```diff
@@ -61,7 +75,7 @@ const Subscription: React.FC<Props> = ({ subscription }) => {
       ) : (
         <>
           <p>{t('user:payment.no_subscription_new_payment')}</p>
-          <Button toModal="choose-offer" label={t('user:payment.view_plans')} />
+          <Button onClick={onClick} label={t('user:payment.view_plans')} />
```

Comment:
`onClick` shouldn't be needed, but it's fine for now:

```tsx
<Button to={modalURLFromLocation(location, 'choose-offer')} label={t('user:payment.view_plans')} />
```

Reply:
The `onClick` handler enabled us to reflect on the `isLoading` state and render a `LoadingOverlay` while the URL is being fetched and we're waiting for the redirection to occur.

Reply:
Sorry, my previous answer is not the correct one for this use case. You're actually right, I'm changing it right now.

Reply:
Sorry, my previous answer to this is incorrect for this use case. You're actually right, with the addition that `modalURLFromLocation` has to be used because we're navigating to a modal (`&u=choose-offer`), not an actual route (`/choose-offer`).

Done.